bundles/net.tourbook/src/net/tourbook/ui/tourChart/TourChart.java

```diff
@@ -1450,6 +1464,52 @@ private void createActions_10_GraphActions() {
             Messages.Image__Graph_Swim_Swolf,
             Messages.Image__Graph_Swim_Swolf_Disabled,
             GRAPH_CONTRIBUTION_ID_SWIM_SWOLF);
+
+      /*
+       * CUSTOM TRACKS
+       */
+      createActions_10_GraphActionsCustomTracks();
+   }
+
+   /**
+    * Create all graph actions for CUSTOM TRACKS
+    */
+   private void createActions_10_GraphActionsCustomTracks() {
```

Comment:
Custom track actions are displayed after the default actions, so should also be the method sorting

![mt-method sorting](https://user-images.githubusercontent.com/1283445/108962109-8f769b00-7678-11eb-8233-61cf48a4fd54.png)